### PR TITLE
seori/DSODA-295

### PR DESCRIPTION
**Pull request by msbeethoven**

### CODE REVIEW

**Comment by jkeen** [comment]
I'm not sure It's supposed to look like [https://zpl.io/VOpEQWL](this), yeah? 
image


Because right now it looks like this: 
image

Or is some of this in flux still?

**Comment by msbeethoven** [comment]
> I'm not sure if you were looking for Visual QA, but it's supposed to look like [https://zpl.io/VOpEQWL](this), yeah?
> image
> 
> Because right now it looks like this:
> image
> 
> Or is some of this in flux still?

Because the  host images, show details, currently playing, and playlist history are not showing up, I wanted to just get this out since I'm not 100% sure that it will look like this when we get those other components in, especially when it comes to the overlapping host image on the currently playing div.

**Comment by jkeen** [comment]
Oh wait, looks like some tests are failing? I think we lost some data from some tests I added in another commit on master. Go check the diff and make sure the "data-test-" attributes are preserved in what you've got

File: app/templates/listen.hbs [hunk 1]
```diff
@@ -1,96 +1,90 @@
...
 
-  {{#liquid-if (or this.nowPlaying.hasCurrentTrack (not (is-empty this.recentlyPlayed)))}}
+  <div class="playlist-history-container">
+    {{#liquid-if (or this.nowPlaying.hasCurrentTrack (not (is-empty this.recentlyPlayed)))}}
     <div class="playlist-history">
       {{#if this.nowPlaying.track}}
-        <div class="currently-listening-to">
-          <div class="currently-listening-to-label">Currently Playing</div>
-          <TrackInfo
-            data-test-component="current-track"
```
Comment (jkeen) [inline app/templates/listen.hbs]: We need this line

File: app/templates/listen.hbs [hunk 2]
```diff
@@ -1,96 +1,90 @@
...
       {{/if}}
 
-      {{#if (not (is-empty this.recentlyPlayed))}}
-        <div class="recently-played">
-          <div class="recently-played-label">Recent Music</div>
-          <div class="recently-played-song-list">
-            {{#each (take 3 (filter-by "isLive" false this.recentlyPlayed)) as |track index|}}
-              <TrackInfo
-                data-test-component={{concat "recent-track-" (inc index 1)}}
-                @trackTitle={{track.title}}
```
Comment (jkeen) [inline app/templates/listen.hbs]: and that one ^

**Pull request merged**